packages/view-types/feature-list/src/FeatureListOptions.js

```diff
@@ -35,62 +35,69 @@ export default function FeatureListOptions(props) {
     <OptionsContainer>
       {children}
       <TableRow>
-        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-option-select">
+        <TableCell className={classes.labelCell} variant="head" scope="row">
           Sort Ordering
         </TableCell>
-        <TableCell>
+        <TableCell variant="body">
           <OptionSelect
+            key="feature-list-sort-option-select"
             className={classes.select}
             value={featureListSort}
             onChange={handleFeatureListSortChange}
             inputProps={{
-              id: 'feature-list-sort-option-select',
+              id: 'feature-list-sort-option',
             }}
           >
             {FEATURELIST_SORT_OPTIONS.map(option => (
-              <option key={option} value={option}>{option}</option>
+              <option id={option} value={option}>{option}</option>
```

Comment:
```suggestion
              <option key={option} value={option}>{option}</option>
```

The key here is because we are using `.map` - please see https://react.dev/learn/rendering-lists#rules-of-keys